site/docs/foundations/spacing.mdx

```diff
@@ -126,6 +126,6 @@ For spacing purposes, separators are considered to be a type of content and ther
 
 Only use separators if the content cannot be effectively divided in another way. Too many repeating lines can overload a design and create unnecessary visual noise.
 
-The [Division](division) Foundation provides further guidance on the use of separators and their alternatives.
+The [Division](/salt/foundations/division) Foundation provides further guidance on the use of separators and their alternatives.
```

Comment:
We refer to 'separators' as a 'divider' in our Salt component library, and since dividers are intended to be used to create 'separation', do we need to make this more clear? I've seen designers create their own separators using the line tool and applying the separable characteristic / design token to it... what is the best guidance we can offer here to make it clear how to apply this concept? Do we want to point to the divider component from here? Or simply rename the 'Separators' section to 'Dividers' and reword this section?

Reply:
@amritadesmet to update in foundation rework